tests/integration/test_system_containers_rpm.sh

```diff
@@ -149,3 +150,11 @@ for i in /usr/local/lib/renamed-atomic-test-system-hostfs /usr/local/lib/secret-
 do
     test -e $i
 done
+
+echo "This message will not be deleted" > /usr/local/lib/secret-message
+
+ATOMIC_OSTREE_TEST_FORCE_IMAGE_ID=NEW-ID ${ATOMIC} containers update atomic-test-system-hostfs
+
```

Comment:
Minor: Do you think it is better to also include a check to see if the error message is there to notify the user? 
I.e: We check if `Do not delete xxx as it was manually modified`  is outputted?

Reply:
I do that no?

There is:

```
+assert_matches "This message will not be deleted" /usr/local/lib/secret-message
```